airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -530,6 +563,38 @@
         }
     }
 
+    function loadMediaElementPlayer(song, position) {
+        var player = $('#audioPlayer').get(0);
+
+        // Is this a new song?
+        if (player.src != song.streamUrl) {
+            // Stop the current playing song and change the media source.
+            player.src = song.streamUrl;
+            // Inform MEJS that we need to load a new media source. The
+            // 'canplay' event will be fired once playback is possible.
+            player.load();
+            // The 'skip' function takes a 'position' argument. We don't
+            // usually send it, and in this case it's better to do nothing.
+            // Otherwise, the 'canplay' event will also be fired after
+            // setting 'currentTime'.
+            if (position && position > 0) {
+                player.currentTime = position;
+            }
+            // Log the song URL.
+            console.log(song.streamUrl);
+
+        // Are we seeking on an already-playing song?
+        } else {
+            // Seeking also starts playing. The 'canplay' event will be
+            // fired after setting 'currentTime'.
+            player.currentTime = position ? position : 0;
```

Comment:
I don't understand this line: If position is non-zero, the use position, otherwise, use zero?
If position can be nil, the javascript-way of casting it to zero would be `player.currentTime = position || 0;`.

Reply:
Ah, thanks! That's what I was going for.